adminapi/cmdline/query.py

```diff
@@ -25,9 +25,11 @@ def main():
 
     attrs = options.attrs if options.attrs else ('hostname', )
```

Comment:
Then it wouldn't be a `tuple`.  A `list` is more appropriate in here anyway, so I am changing it.